articles/native-platforms/android/01-login.md

```diff
@@ -86,48 +92,48 @@ At this point, you're all set to implement the Login in any activity you want.
 
 First, add these lines in the ``onCreate`` method:
 
-```android
+```java
 Auth0 auth0 = new Auth0(${account.clientId}, ${account.namespace});
-            this.lock = Lock.newBuilder(auth0, callback)
+this.lock = Lock.newBuilder(auth0, callback)
                     // Add parameters to the Lock Builder
-                    .build();
+				.build();
 ```
 
 Second, add these lines in the ``onDestroy`` method:
 
-```android
+```java
  protected void onDestroy() {
-            super.onDestroy();
-            // Your own Activity code
-            lock.onDestroy(this);
-            lock = null;
-        }
+	super.onDestroy();
+	// Your own Activity code
+	lock.onDestroy(this);
+	lock = null;
+}
 ```
 Third, add the authentication callback, inside your activity:
 
-```
+```java
 private LockCallback callback = new AuthenticationCallback() {
-            @Override
-            public void onAuthentication(Credentials credentials) {
-				// Login Success response
-            }
-
-            @Override
-            public void onCanceled() {
-				// Login Cancelled response
-            }
-
-            @Override
-            public void onError(LockException error){
-				// Login Error response
-            }
-        };
+	@Override
+	public void onAuthentication(Credentials credentials) {
+		// Login Success response
+	}
+
+	@Override
+	public void onCanceled() {
+		// Login Cancelled response
+	}
+
+	@Override
+	public void onError(LockException error){
+		// Login Error response
+	}
+};
 ```
 
 Finally, whenever you want to start the login widget, call:
 
-```
-            startActivity(this.lock.newIntent(this));
+```java
+startActivity(this.lock.newIntent(this));
 
 ```
 
```

Comment:
@tenmaster Since you are editing this doc please fix these as well:

This `[![Lock.png](/media/articles/libraries/lock-android/login.png)]`
Should be `![Lock.png](/media/articles/libraries/lock-android/login.png)`

Also, the image size is 540 × 960. The height is too much and it does not display nice. Make the image smaller please.